src/ripple/core/DatabaseCon.h

```diff
@@ -89,6 +90,22 @@ class DatabaseCon
         Config::StartUpType startUp = Config::NORMAL;
         bool standAlone = false;
         boost::filesystem::path dataDir;
+        // If unseated, then the `globalPragma` are not used,
+        // otherwise should point to `globalPragma`
+        std::shared_ptr<std::vector<std::string> const> commonPragma;
```

Comment:
Yes, I belive I could. This is different than the `bool` param that @nbougalis objected to in https://github.com/ripple/rippled/pull/3429#discussion_r435656253, because it's a member with a clearer purpose. I can also abstract that away so the caller only sees a function call.